ivy/functional/ivy/utility.py

```diff
@@ -147,92 +147,113 @@ def any(
+        Output array, for writing the result to. It must have a shape that the inputs
+        broadcast to.Default: None.
 
     Returns
     -------
-    ret
-        if a logical OR reduction was performed over the entire array, the returned
+    ret : ivy.Array
+        If a logical OR reduction was performed over the entire array, the returned
         array must be a zero-dimensional array containing the test result; otherwise,
         the returned array must be a non-zero-dimensional array containing the test
-        results. The returned array must have a data type of ``bool``.
-
-
-    This method conforms to the `Array API Standard
-    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
-    `docstring <https://data-apis.org/array-api/latest/
-    API_specification/generated/array_api.any.html>`_
-    in the standard.
-
-    Both the description and the type hints above assumes an array input for simplicit
-    y,but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
-    instances in place of any of the arguments.
+        results. The returned array must have a data type of bool.
+
+    Raises
+    ------
+    IvyIndexError
+        If the provided axis is invalid.
+
+    Notes
+    -----
+    - Positive infinity, negative infinity, and NaN must evaluate to True.
+      Explanation: The logical OR operation considers positive infinity, negative
+      infinity, and NaN as True.
+
+    - If x is an empty array or the size of the axis (dimension) along which to
+      evaluate elements is zero,the test result must be False.
+      Explanation: If the input array x is empty or the axis along which the elements
+      are evaluated has a size of zero,the result of the logical OR operation must be
+      False.
+
+    This method conforms to the Array API Standard.
+    Both the description and the type hints above assume an array input for simplicity,
+    but this function is nestable, and therefore also accepts Ivy Container instances
+    in place of the arguments.
 
     Examples
     --------
-    With :class:`ivy.Array` input:
-
-    >>> x = ivy.array([2, 3, 4])
-    >>> y = ivy.any(x)
-    >>> print(y)
-    ivy.array(True)
-
-    >>> x = ivy.array([[0],[1]])
-    >>> y = ivy.zeros((1,1), dtype='bool')
-    >>> a = ivy.any(x, axis=0, out = y, keepdims=True)
-    >>> print(a)
-    ivy.array([[True]])
-
-    >>> x=ivy.array(False)
-    >>> y=ivy.any(ivy.array([[0, 3],[1, 4]]), axis=(0,1), out=x, keepdims=False)
-    >>> print(y)
-    ivy.array(True)
-
-    >>> x=ivy.array(False)
-    >>> y=ivy.any(ivy.array([[[0],[1]],[[1],[1]]]),axis=(0,1,2), out=x, keepdims=False)
-    >>> print(y)
-    ivy.array(True)
-
-    With :class:`ivy.Container` input:
-
-    >>> x = ivy.Container(a=ivy.array([0, 1, 2]), b=ivy.array([3, 4, 5]))
-    >>> y = ivy.any(x)
-    >>> print(y)
-    {
-        a: ivy.array(True),
-        b: ivy.array(True)
-    }
+    Example 1:
+    >>> import ivy
```

Comment:
The former way of enlisting examples has been consistent across the entirety of ivy documentation, if you think this can be improved, please file a new issue. For now please revert these changes